base/src/main/java/com/smartdevicelink/proxy/rpc/OnAppCapabilityUpdated.java

```diff
@@ -0,0 +1,53 @@
+package com.smartdevicelink.proxy.rpc;
```

Comment:
@kostyaBoss I generated this file using the code generator tool, and see a few differences between the file I am generating and yours.

Can you regenerate this?  An extra step here is under the `<enum name=FunctionID>` tag you will need to define an element like this `<element name="OnAppCapabilityUpdatedID" value = "32791" hexvalue="8017" since="7.1"/>` around line 3082 in the `MOBILE_API.xml`.

Reply:
@santanamk I've tried this but didn't work for me. Will try rolling back the data manually

Reply:
@kostyaBoss  I generated the `OnAppCapabilityUpdated.java` file using your generator tool. There are a lot of differences between the generated `OnAppCapabilityUpdated.java` file and the one you are using. I have attached the generated `OnAppCapabilityUpdated.java` file 
[OnAppCapabilityUpdated.txt](https://github.com/smartdevicelink/sdl_java_suite/files/5780103/OnAppCapabilityUpdated.txt)
here. It is a .txt file but you can rename the extension to .java.